src/libraries/System.Text.Json/gen/JsonSourceGenerator.Parser.cs

```diff
@@ -104,6 +106,14 @@ public Parser(KnownTypeSymbols knownSymbols)
                     return null;
                 }
 
+                LanguageVersion? langVersion = _knownSymbols.Compilation.GetLanguageVersion();
```

Comment:
I believe source generators are only supported in C#. Technically a `Compilation` instance could be a Visual Basic compilation but I don't believe it's possible here. I thought it made sense to account for that case regardless, hence the nullable version.

Reply:
To answer your actual question, the `LanguageVersion` enum is C# specific since it's in the `CSharp` namespace: https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.languageversion?view=roslyn-dotnet-4.6.0